app/src/main/res/anim/expand.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Reminder that this file needs to be named more specifically.

Also, have you adjusted this animation to better match the mocks?

Reply:
I have verified with @mschanteltc that the current animation looks good.